### ISSUE TITLE
Ensure system is readable if core's dark mode theme is enabled

### ISSUE THREAD

**Issue by secondworldpublishing** [koboldpress/black-flag#787]
The styling of the RTE for the last few versions is far too dark for the user to read efficiently. 

![Screenshot 2024-10-08 at 8 24 04 AM](https://github.com/user-attachments/assets/09a04793-bd05-4b74-ada0-a5ea799dd6e0)

**Comment by arbron** [comment]
No work has been done to support dark mode yet.

**Comment by secondworldpublishing** [comment]
![image](https://github.com/user-attachments/assets/a02e9bc4-7d86-434e-9577-4e7194668872)

Apologies.  I forgot that I had dark mode activated in my OS.
